src/docs/patterns/forms.mdx

```diff
@@ -160,54 +174,36 @@ For screens such as email verification where there is a single field with a know
     }
 />
 
-When the user attempts to submit the form by clicking on the submit button on their screen (desktop) or on their software keyboard (mobile), all the invalid fields should be highlighted with red error text.
-
-<ImageGrid
-    images={[dismissKeyboardErrorAutoscrollImage, dismissKeyboardErrorTextImage]}
-    caption={[
-        'Left: The UI should autoscroll to and focus on the first invalid field.',
-        'Right: On a short form, you may choose to display a descriptor on the top of the form instead.',
-    ]}
-/>
-
 ### Disabling the Submit Button
 
-In most cases, the submit button **should not** be disabled - users should not be forced to search for their mistake.
-
-<!--
-TODO:
-This image is conveying conflicting idea with the two gifs above,
-either remvoe it, or find a better way to picture this problem (like those do's and dont's on Material Design)
-<ImageGrid
-    images={[SubmitButtonOnClickImage]}
-    regularWidth
-    caption={
-        'Do not disable the submit button. Instead, use error text to highlight the invalid input after the user clicks the button.'
-    }
-/>
--->
-
-> **Exception**: on desktop applications, when there are fewer than four text fields, and all the fields are verified on change, we recommend you disable the submit button to indicate that there are some errors in the user input (invalid value, missing a required field).
+If there are errors indicated on any of the form fields, the submit button should be disabled. If you are performing verification when clicking the submit button (onAction), all the invalid fields should be indicated (if they aren't already).
 
 <React.Fragment>
-    <ImageGrid
+    {/* <ImageGrid
         images={[SubmitButtonOnChangeImage]}
         regularWidth
         caption={
             'When your input field field is verified on change, disable the submit button until the user enters a valid input.'
         }
-    />
+    /> */}
     <ImageGrid
         images={[SubmitButtonOnBlurImage]}
         regularWidth
         caption={'The submit button is disabled until the user has entered something for all the required fields.'}
     />
+    <ImageGrid
+        images={[dismissKeyboardErrorAutoscrollImage, dismissKeyboardErrorTextImage]}
+        caption={[
+            'Left: The UI should auto-scroll to and focus on the first invalid field.',
+            'Right: On login screens, form fields should not shift when errors appear.',
```

Comment:
Just a note that these ImageGrids get displayed in a stacked way on mobile, and it would make no sense to call them "left" and "right"